### PR TITLE
fix bleeding

### PR DESCRIPTION
**Pull request by VernaSaurus**
see title

CL: [Fix] fixed bleeding 5x more than you should
CL: [Fix] fixed continuing to bleed forever until you apply gauze

### CODE REVIEW

**Review by Bod9001 — DISMISSED** [review]

File: UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs [hunk 1]
```diff
@@ -182,7 +182,7 @@ public void ImplantPeriodicUpdate()
 			if(IsBleeding)
 			{
 				InternalBleedingLogic();
-				HealthMaster.CirculatorySystem.Bleed(limbLossBleedingValue);
+				//HealthMaster.CirculatorySystem.Bleed(limbLossBleedingValue); causes you to continue bleeding after hitting 0 bleedstacks
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs]: please don't comment out code, remove it.

File: UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs [hunk 2]
```diff
@@ -333,12 +333,14 @@ public void BodyPartRemoveHealthMaster()
 		/// </summary>
 		public void TryRemoveFromBody(bool beingGibbed = false)
 		{
+			bool alreadyBleeding = false;
 			SetRemovedColor();
 			foreach (var bodyPart in HealthMaster.BodyPartList)
 			{
-				if (bodyPart.BodyPartType == BodyPartType.Chest)
+				if (bodyPart.BodyPartType == BodyPartType.Chest && alreadyBleeding == false)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs:339]: isn't ``alreadyBleeding`` always ``false``?

File: UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs (hunk 1 above)
Comment (VernaSaurus) [reply UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs]: gotcha

File: UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs (hunk 2 above)
Comment (corp-0) [reply UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs:339]: Aah, I understand it now.

Comment (VernaSaurus) [reply UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs:339]: nope, for some reason BodyPartType.Chest is found five times, and adding this check causes it to only make bleeding happen once

**Review by corp-0 — APPROVED** [review]

**Review by NoooneyDude — APPROVED** [review]

**Pull request merged**